src/Workspaces/Core/Portable/Workspace/FileTextLoader.cs

```diff
@@ -50,7 +50,7 @@ public class FileTextLoader : TextLoader
         /// <param name="path">An absolute file path.</param>
         /// <param name="defaultEncoding">
         /// Specifies an encoding to be used if the actual encoding can't be determined from the stream content (the stream doesn't start with Byte Order Mark).
-        /// If not specified auto-detect heuristics are used to determine the encoding. If these heuristics fail the decoding is assumed to be <see cref="Encoding.Default"/>.
+        /// If not specified auto-detect heuristics are used to determine the encoding.
```

Comment:
Not by this type. The logic that determines the encoding used when it isn't specified is different between the two ITextFactoryService implementations that might be used. It appears that this comment is true if its the service from the editor layer, but not necessarily true in the Workspaces layer, where Encoding.Default is not available. So, at the very least, this XML doc comment was misleading.

I ended up removing it because the CREF for Encoding.Default no longer resolves after moving FileTextLoader to the Workspaces layer.